syntax = "proto3";

option go_package = "pb/game";

message Vector3
{
	double x = 1;
	double y = 2;
	double z = 3;
}


message PlayerInt32Array
{
	repeated int32 int32_valuse = 1;
}

message PlayerUInt32Array
{
	repeated uint32 uint32_valuse = 2;
}

message PlayerInt64Array
{
	
}

message PlayerUInt64Array
{
	
}

